packages/core/src/components/sections/Hero/Hero.tsx

```diff
@@ -59,7 +59,7 @@ const Hero = ({
             alt={image.alt}
             width={360}
             height={240}
-            sizes="(max-width: 360px) 50vw, (max-width: 768px) 90vw, 50vw"
+            sizes="(max-width: 360px) 40vw, (max-width: 768px) 90vw, 50vw"
```

Comment:
question: could you explain why this was changed to 40?

Reply:
I forgot to change the max width to 412px (updated [here](https://github.com/vtex/faststore/pull/2470/commits/9f14120732793077d0d41036df33ed118044228d)). Well, that's because we sometimes get a 'not so good' LCP related to this image in the PSI. PSI recommends using a small image there. Changing it to 40vw is forcing the browser to download a small image.

Reply:
Got it! Thanks